uiplib/gui/settingsTab.py

```diff
@@ -147,6 +148,21 @@ def handle_settings(appObj):
     update_settings(appObj.new_settings)
 
 
+def update_settings_view(appObj, mainFrame):
```

Comment:
hey so in update settings, the new_settings would be a parameter so we can call it with different settings sometime in the futiure